tests/cpp/test_matmul.cpp

```diff
@@ -3775,12 +3800,28 @@ TEST_F(HopperMatmulTest, HSH_NT_128BSwizzle) {
     tv2->axis(-3)->parallelize(ParallelType::Mma);
   }
 
-  for (auto tv : {tv3c, tv3}) {
+  if (!use_smem_epilogue) {
+    for (auto tv : {tv3c, tv3}) {
+      auto s = mma_utils::MmaSwizzler::scheduleMmaOutputAllocation(
+          tv->getLoopDomain());
+      tv->setLoopDomain(s.as<IterDomain*>());
+    }
+  } else {
     auto s = mma_utils::MmaSwizzler::scheduleMmaOutputAllocation(
-        tv->getLoopDomain());
-    tv->setLoopDomain(s.as<IterDomain*>());
+        tv3c->getLoopDomain());
+    tv3c->setLoopDomain(s.as<IterDomain*>());
+    tv3c->setAllocationDomain(s.as<IterDomain*>(), true);
+
+    // We need to split and merge dimensions which have
+    // been marked parallel/vectorized. So we revert the
+    // inner-most dimension back to serial.
+    tv3->axis(-1)->parallelize(ParallelType::Serial);
```

Comment:
How does tv3 look before this line? I expected it to just be block tiled and the outer dims parallelized BIDy and BiDy by the propagation from tv2.

Reply:
I am not sure what I was doing last night that I saw a vectorized dim there! 
There isn't one - will remove the serial type.